/**
 * Returns the exception that occurred during a static initialization that
 * caused this error to be created.
 *
 * <p>This method predates the general-purpose exception chaining facility.
 * The {@link Throwable#getCause()} method is now the preferred means of
 * obtaining this information.
 *
 * @return the saved throwable object of this
 *         <code>ExceptionInInitializerError</code>, or <code>null</code>
 *         if this <code>ExceptionInInitializerError</code> has no saved
 *         throwable object.
 */
public Throwable getException() {
    return exception;
}